### PR TITLE
Use dcos-check-runner for postflight checks

### PR DESCRIPTION
**Pull request by branden**
## High-level description (required)

This updates the postflight script to use the dcos-check-runner added with https://github.com/dcos/dcos/pull/2921.

## Corresponding tickets (required)

These JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-3559](https://jira.mesosphere.com/browse/DCOS_OSS-3559) Update dcos-launch to use dcos-check-runner for postflight checks

## Related tickets (optional)

Other tickets related to this change

  - [DCOS_OSS-3491](https://jira.mesosphere.com/browse/DCOS_OSS-3491) Separate check runner from dcos-diagnostics

## Related PRs (optional)

Is this change going to be propagated up into dcos-integration-tests in dcos/dcos or another repo? Does this change require changes in dcos-test-utils? Link the corresponding PRs here:

https://github.com/dcos/dcos/pull/2921

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] **Added or updated any relevant documentation**

[Integration tests](https://teamcity.mesosphere.io/project.html?projectId=DcosIo_DcosLaunch&branch_DcosIo_DcosLaunch=%3Cdefault%3E) were run and

**dcos/dcos master** (checks provided by dcos-diagnostics)
  - [x] AWS Cloudformation Simple (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096234&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_Aws)
  - [x] Azure Resource Manager (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096236&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_AzureResourceManager)
  - [x] Onprem-AWS (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096238&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_OnpremAwsNew)
  - [x] Onprem-GCE (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096240&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_OnpremGce)

**dcos/dcos#2921** (checks provided by dcos-check-runner)
  - [x] AWS Cloudformation Simple (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096242&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_Aws)
  - [x] Azure Resource Manager (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096252&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_AzureResourceManager)
  - [x] Onprem-AWS (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096246&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_OnpremAwsNew)
  - [x] Onprem-GCE (link to job: https://teamcity.mesosphere.io/viewLog.html?buildId=1096248&tab=buildResultsDiv&buildTypeId=DcosIo_DcosLaunch_IntegrationTests_OnpremGce)

**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

### CODE REVIEW

**Comment by branden** [comment]
@margaret @cprovencher This is ready for review. Please take a look, thanks!

**Review by orsenthil — APPROVED** [review]
LGTM.

File: dcos_launch/platforms/onprem.py [hunk 1]
```diff
@@ -238,30 +238,51 @@ def do_postflight(client: ssh_client.AsyncSshClient):
...
+    max_runs=900
+
+    runs=$max_runs
+    until out=$($cmd) || [[ runs -eq 0 ]]; do
+        sleep 1
+        let runs=runs-1
+    done
+    retcode=$?
+
+    echo "$out"
```
Comment (margaret) [inline dcos_launch/platforms/onprem.py:254]: Would it be helpful to also print how many times it had to try before it worked?

**Review by margaret — COMMENTED** [review]
Not going to block on echoing `runs`. Looks good overall, thanks for pushing that update down to dcos-launch.
Edit to add: waiting for Charles to take a look before merging

**Review by cprovencher — APPROVED** [review]

File: dcos_launch/platforms/onprem.py (hunk 1 above)
Comment (branden) [reply dcos_launch/platforms/onprem.py:254]: I think it's more useful to know how much time elapsed before checks started passing. That can be inferred from timestamps in logs, so I wasn't too worried about adding that info here.

**Pull request merged**